app.py

```diff
@@ -30,11 +29,67 @@ def send_mail(form_msg, form_name, form_email):
         # print("sent")
     except urllib.HTTPError:
         print("not sent")
+        slack_notifier(mode=1)
+    except Exception:
+        print("Some other exception occured. Not sent")
+        slack_notifier(mode=2)
+
+    from_email = Email(os.environ['KOSS_EMAIL'])
+    # print("email init")
+    to_email = Email(form_email)
+    # print("email init2")
+    subject = "Query Recieved"
+    # print("subject")
+    content = Content("text/plain", "Hi "+form_name+",\n\n"+"Your message has been recieved by us and we \
+                       will respond to it soon."+"\nThank-you for communicating with us, hope your query \
+                       will cleared by our team."+"\n\n\nKOSS IIT Kharagpur")
+    # print("content")
+    mail = Mail(from_email=from_email, subject=subject,
+                to_email=to_email, content=content)
+    # print("mail init")
+    try:
+        response = sg.client.mail.send.post(request_body=mail.get())
+        # print("sent")
+    except urllib.HTTPError:
+        print("not sent")
+        slack_notifier(mode=1)
     except Exception:
         print("Some other exception occured. Not sent")
+        slack_notifier(mode=2)
+
+    slack_notifier(form_msg, form_name, form_email)
     return None
 
 
+# slack notifier module
+def slack_notifier(form_msg="", form_name="", form_email="", mode=0, count=0):
+    headers = {"Content-Type": "application/json"}
+
+    if mode == 0:
+        data = json.dumps({"text": "Query from "+form_name+"<"+form_email+
+                           ">\n\n"+"QUERY : "+form_msg+"\n\nPlease respond \
+                           soon."})
```

Comment:
This will have a huge space between `Please respond` and `soon`. Use a format method of string. something like this.
```
data = json.dumps({"text": ("Query from {}<{}>\n\nQUERY : {}\n\nPlease respond"
                            " soon.".format(form_name,form_email,form_msg))})
```